src/components/context_menu/context_menu_item.stories.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { disableStorybookControls } from '../../../.storybook/utils';
+
+import {
+  EuiContextMenuItem,
+  EuiContextMenuItemProps,
+} from './context_menu_item';
+
+const meta: Meta<EuiContextMenuItemProps> = {
+  title: 'EuiContextMenuItem',
+  component: EuiContextMenuItem,
+  argTypes: {
+    ...disableStorybookControls(['buttonRef']),
+    icon: { control: 'text' },
```

Comment:
This might be a better UI:

```suggestion
    icon: { control: 'text' },
    toolTipTitle: { control: 'text' },
    toolTipContent: { control: 'text' },
```

Reply:
I'm deliberately omitting those two props since they're being deprecated in a month's time!

Reply:
For `toolTipContent`, I'd rather leave it as is - I'm not totally convinced yet that _all_ `ReactNode` types should show up as text controls. Leaving it as a 'raw' control lets people type in, e.g. `9` or `false` and have that be evaluated as their native types instead of being coerced to a string.

IMO, ideally we'd have a custom control that lets people type in, e.g. `<div>whatever</div>` which gets eval'd to JSX, but I can understand why that would be spicy (I've seen React Styleguidist do this, so I know it's possible, but it's not something Storybook supports or has indicated they will support).

Reply:
> I'm deliberately omitting toolTipTitle and toolTipPosition as props

Oh ok, they were still showing up as configurable in the UI, which is why I asked.